NuGet.config

```diff
@@ -12,6 +12,7 @@
     <!--  Begin: Package sources from dotnet-runtime -->
     <!--  End: Package sources from dotnet-runtime -->
     <!--  Begin: Package sources from dotnet-templating -->
+    <add key="darc-pub-dotnet-templating-385d77b" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-385d77bd/nuget/v3/index.json" />
```

Comment:
@v-wuzhai why do we need this 8.0 publishing feed in main?

Reply:
We can see that Microsoft.TemplateEngine.Authoring.Tasks: 8.0.100 is not in nuget feeds from the error message https://dev.azure.com/dnceng-public/public/_build/results?buildId=457466&view=logs&j=fa59fe4e-195c-56fa-189b-58fd241f10dd&t=e0ea3df6-0956-547c-f7b9-9062676d48ed&l=197.